tests/Line.php

```diff
@@ -136,3 +136,9 @@
 
     expect($line->toString())->toBe('<bg=default;options=>String Titlecase</>');
 });
+
+it('sets the text in snakecase', function () {
+    $line = line('SnakeCase snakeCase snakeCASE SNAKECase')->snakecase();
```

Comment:
You can you separate this into two lines like the other tests?

Reply:
Done 😅